### PR TITLE
Add OpenCR servo ROS node

### LINKED ISSUE #24: Attach gripper to TurtleBot.
**Describe the task**
Attach gripper.

**Depending on**
Nothing.

**Actions**
Find and print model.

### PR DESCRIPTION
**Pull request by DylanVanAssche**
This PR adds the OpenCR Arduino code to control a servo through the ROS network.
The servo reads the data as a `std_msgs Byte` on `/grabber`.
Fixed #24

### CODE REVIEW

**Comment by DylanVanAssche** [comment]
Ready for review!

**Comment by 4ilo** [comment]
We made some errors in making this driver, the basic motor controls are no longer working.

**Comment by DylanVanAssche** [comment]
> We made some errors in making this driver, the basic motor controls are no longer working.

Oh, that's a bummer :(
Lets investigate this and fix it next week!

**Comment by DylanVanAssche** [comment]
Done!

File: opencr/servo_ros/servo_ros.ino [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#include <ros.h>
+#include <std_msgs/Byte.h>
+#include <Servo.h>
+#define SERVO_PIN 9
+#define SERVO_MIN 0
+#define SERVO_MAX 180
```
Comment (4ilo) [inline opencr/servo_ros/servo_ros.ino]: change to 170

**Review by 4ilo — APPROVED** [review]

**Pull request merged**